spring-kafka/src/main/java/org/springframework/kafka/support/serializer/JsonSerializer.java

```diff
@@ -113,9 +113,9 @@ public void setTypeMapper(Jackson2JavaTypeMapper typeMapper) {
 	 * @since 2.1.3
 	 */
 	public void setUseTypeMapperForKey(boolean isKey) {
-		if (!this.typeMapperExplicitlySet
-				&& this.getTypeMapper() instanceof AbstractJavaTypeMapper) {
-			((AbstractJavaTypeMapper) this.getTypeMapper()).setUseForKey(isKey);
+		if (!this.typeMapperExplicitlySet && getTypeMapper() instanceof AbstractJavaTypeMapper) {
+			((AbstractJavaTypeMapper) getTypeMapper())
+					.setUseForKey(isKey);
```

Comment:
I prefer to have
methods()
.called()
.as()
.a()
.chain()

In this case we can `getTypeMapper().setUseForKey(isKey)` essentially. So, since we call two methods, that's already a chain.

Reply:
In general, yes, but I think it's overkill when there's only one `.`

Reply:
Well, when you add more methods to this chain, you would need to place this single `.` on a new line, too. Otherwise it won't look like a smooth chain.
So, a change offset is going to be bigger the next time.

But since you still don't like it, feel free to revert on merge and let's go ahead with other issue to fix!
No reason to argue just for one dot. 😄

Reply:
I don't feel that strongly; merging...